### PR TITLE
Scss support added + BC on output definition

### PR DESCRIPTION
**Pull request by kptLucek**
@bronek89

### CODE REVIEW

File: tasks/lib/flaticon.js [hunk 1]
```diff
@@ -6,137 +6,145 @@
  */
 
 module.exports = (function () {
-    'use strict';
-
```
Comment (bronek89) [inline tasks/lib/flaticon.js]: Why did you change spaces to tabs in indentation?

**Comment by kptLucek** [comment]
@bronek89 fixed :)
